setup.cfg

```diff
@@ -1,5 +1,4 @@
 [tool:pytest]
-addopts = -vs --reuse-db --showlocals --tb=short
```

Comment:
why is this being removed - we need this for normal testing, right?

Reply:
Maybe this was by error, I'll recheck.

Reply:
Agreed, that change should be backed out, we need that for our regular tests. The test results on travis are useless without those options.